CHANGELOG.md

```diff
@@ -0,0 +1,54 @@
+# Changelog
+
```

Comment:
unreleased should be at the top ?

Reply:
I disabled unreleased from the Changelog since the manual entries have it, and felt wrong to have it twice. It's included down inside the notables changes section.